src/content/docs/en/reference/container-reference.mdx

```diff
@@ -40,21 +40,21 @@ Enables rendering components using [HTML streaming](/en/guides/server-side-rende
 
 #### `renderers` option
 
-**Type:** `AstroRenderer[]`
+**Type:** `SSRLoadedRendere[]`
 
-A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script
+A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script.
+
+In a vite-full environment, renderers should be loaded by using the function `loadRenderers`, from the virtual module `astro:container`.
 
 The following example provides the necessary object to render an Astro component that renders a React component: 
 
 ```js
+import { getContainerRenderer } from "@astrojs/react";
```

Comment:
```suggestion
import { getContainerRenderer } from "@astrojs/react";
```
I'm assuming this is what you import from each different renderer integration. What happens when you have multiple to render (e.g. from `@astrojs/vue` or `@astrojs/svelte` at the same time? Are they all called exactly the same thing, and is that OK? (No naming collision?)

We do say above you need to do this "for each framework rendered" and I'm not immediately sure what that would look like if you tried to render two frameworks. Maybe the example below could be updated to show multiple? (I think then, it will be pretty easy for the reader to visualize what only one would look like.)